test/Driver/batch_mode_with_supplementary_filelist_unicode.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %empty-directory(%t)
+// RUN: echo 'print("Hello, World!")' >%t/main.swift
+// RUN: touch %t/𝔼-file-01.swift %t/😂-file-02.swift %t/Ω-file-03.swift
+//
+// If the supplementary output file map does not escape the characters in the
+// source files, the frontend won't recognize the desired outputs.
+//
+// RUN: (cd %t && %target-build-swift -c -save-temps -emit-dependencies -emit-loaded-module-trace -emit-loaded-module-trace-path %t/lmt -serialize-diagnostics -driver-use-filelists -j2 main.swift  𝔼-file-01.swift 😂-file-02.swift Ω-file-03.swift  -module-name mod)
```

Comment:
What's the point of using `-emit-loaded-module-trace-path` if you don't test the output? And doesn't that only work in single-compile mode anyway?

Reply:
This shouldn't need `-save-temps` either; there are no temporaries.